website/docs/integrations/amplitude.mdx

```diff
@@ -18,13 +18,13 @@ There are two available integration opportunities between ConfigCat and Amplitud
 
 Every feature flag change in ConfigCat is annotated on the Amplitude charts as a vertical line and some details are added automatically about the change.
 
-<img src="/docs/assets/amplitude_chart.png" className="bordered-img zoomable" alt="amplitude_chart" />
+<img src="/docs/assets/integrations/amplitude/overview.png" alt="ConfigCat Amplitude Integration Overview" decoding="async" loading="lazy"/>
 
 ### Installation
 
 1. Have an <a href="https://www.amplitude.com/" target="_blank">Amplitude subscription.</a>
 2. Get an <a href="https://www.docs.developers.amplitude.com/analytics/find-api-credentials/" target="_blank">Amplitude API Key and Secret Key.</a>
-   <img src="/docs/assets/amplitude_apikey_secretkey.png" className="zoomable" alt="amplitude_apikey_secretkey" />
+   <img src="/docs/assets/integrations/amplitude/setup.png" alt="ConfigCat Amplitude Integration Setup" decoding="async" loading="lazy"/>
```

Comment:
The `src` URL has changed and the `className` attribute has been removed.

Is this intentional?

Reply:
fixed it